scientific_library/tvb/simulator/simulator.py

```diff
@@ -50,16 +50,24 @@
 from .history import SparseHistory
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray, List, Float
 
+from tvb.simulator.models.reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
+from tvb_multiscale.config import CONFIGURED
```

Comment:
Adding this import, we have a cyclic dependency here (between tvb-library and tvb-multiscale), unhealthy.

Reply:
Reading through that config module, it looks like some boilerplate.  The import here is a convenience to be removed, and those classes could move into tvb's core config as runtime configuration.  This could move into a PR to merge before this one.

Reply:
I removed it.